engine/crates/composition/src/compose/entity_interface.rs

```diff
@@ -0,0 +1,181 @@
+use super::*;
+use crate::composition_ir as ir;
+use std::collections::{BTreeMap, BTreeSet};
+
+pub(crate) fn merge_entity_interface_definitions(
+    ctx: &mut Context<'_>,
+    first: DefinitionWalker<'_>,
+    definitions: &[DefinitionWalker<'_>],
+) {
+    let interface_name = first.name();
+
+    let iface_defs = || definitions.iter().filter(|def| def.kind() == DefinitionKind::Interface);
```

Comment:
Not sure this is a standard abbreviation, personally I'd avoid it for clarity

Reply:
I agree, I use it all the time but it's lazy.